Assets/MixedRealityToolkit.Tools/MSBuild/MSBuildTemplates/SDKProjectTemplate.csproj.template

```diff
@@ -19,6 +19,7 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <!--NOTE: Currently one of the Unity packages has a * in AssemblyVersion identifier in the AssemblyInfo file, so have to do this.-->
     <Deterministic>false</Deterministic>
+	<AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
```

Comment:
It might be best to consistently use spaces for indentation and not mix them with tabs?

Reply:
Oops, good catch.